application/server/notification.go

```diff
@@ -130,6 +115,21 @@ func registerNotifier(c *config.Config, srv types.Server) {
 			// Function blocks on callback, so we need to run it in a separate goroutine
 			go handleShowMessageRequest(srv, params, &logger)
 			logger.Debug().Msg("sending show message request to client")
+		case types.DiagnosticsOverviewParams:
+			logger.Debug().
+				Msgf("received diagnostics overview for %s, discarding", params.Product)
```

Comment:
optional: this could be a nice cleanup, to remove everything related/needed for DiagnosticsOverview